README.md

```diff
@@ -62,9 +62,9 @@ The CI runs unit tests, e2e tests, `clippy` and `cargo fmt`
 
 `cargo test`
 
-### Integration Tests:
+### DB Tests:
```

Comment:
nit: We do still have integration tests (and e2e tests) don't we (should be running as part of CI). Maybe it's enough to link the workflow file but would be nice to include somehow.